website/docs/pipelines/promoting-applications.mdx

```diff
@@ -153,21 +164,27 @@ promotion:
 
 More info in the [spec](../spec/v1alpha1/pipeline/#pipeline)
 
-### Create credentials secret
+### Credentials Secret
+
+In the journey of creating a pull request, there are different secrets involved:
+
+1. Pipeline controller receives events via webhook from leaf clusters. Hmac is used for authN/authZ so a hmac key should be provided in this case.
```

Comment:
Perhaps link to our documentation on setting this up?

Reply:
added